tasks/rslang/speakit.md

```diff
@@ -0,0 +1,126 @@
+- для проверки ссылку на демо-версию приложения необходимо добавить в rs-app
+
+## Технические требования
+- работа приложения проверяется в браузере Google Chrome последней версии
+- не рекомендуется использование jQuery, Angular/React/Vue 
+- рекомендуется использовать webpack, eslint, eslint-config-airbnb-base, babel
+- можно использовать bootstrap, css фреймворки, html и css препроцессоры
+
+## Критерии оценки:
+**Максимальный балл за задание: 100 баллов при кросс-чеке / проверка ментором не предусмотрена**   
+
+### Basic scope +30  
+- [ ] вёрстка, дизайн, UI. Итерактивность кликабельных элементов, различные стили для активного и неактивного состояния. Есть стартовый экран +10
+- [ ] когда приложение открывается, на странице отображается группа из 10 слов, возле каждого слова его транскрипция и иконка аудио. При клике по слову звучит его произношение +10  
+- [ ] при клике по слову изменяется его стиль и отображается соответствующая ему картинка +10
+
+### Advanced scope +60
+- [ ] при клике по слову отображается его перевод +10
+- [ ] если распознавание речи включено, и приложению разрешён доступ к микрофону, произнесённые пользователем слова распознаются и отображаются в текстовом виде +10
+- [ ] проверка правильности произношения происходит путём сравнения распознанного текста с написанием слова. Если слово распознано верно, изменяется стиль его отображения, появляется соответствующее данному слову изображение, результат игры увеличивается на один балл +10
+- [ ] у пользователя есть возможность просмотреть результат игры. Также результат отображается, если все слова распознаны верно. На странице с результатом есть кнопки, позволяющие вернуться к предыдущей игре или запустить новую игру с новым набором слов +10
+- [ ]  на странице результата игры отображаются правильно угаданные слова и слова, в которых были допущены ошибки. Возле каждого слова отображаются его транскрипция, перевод, иконка аудио. По клику по слову звучит его произношение +10
+- [ ] у пользователя есть возможность изменять сложность мини-игры. Всего шесть уровней сложности, которым соответствуют шесть частей коллекции исходных данных  +10
+
+### Hacker scope +10
+- [ ] ведётся долгосрочная статистика мини-игры, можно посмотреть когда (дата и время) и с каким результатом в неё играли. После перезагрузки приложения статистика сохраняется +10
+- [ ] реализован дополнительный, не предусмотренный заданием функционал. Данный пункт не оценивается
+
+## Cross-check
+- инструкция по проведению cross-check: https://docs.rs.school/#/cross-check-flow
+- форма для проверки задания: https://cross-check-speakit.netlify.app/
+
+## Ключевые навыки
+
+- получение данных о словах (предлагаемый набор слов является частью данных, которые будут использованы для финального проекта)
+- работа с асинхронными запросами
+- использование translation API
+- использование Google Web Speech API 
+
+## Видео с разбором задания
+https://youtu.be/qiimg-aIz4w
```

Comment:
Will the overview be uploaded to the RSS YouTube channel? If yes, the link should be updated.